lib/features/sks-menu/presentation/widgets/sks_menu_view_loading.dart

```diff
@@ -0,0 +1,51 @@
+import "package:flutter/material.dart";
+
+import "../../../../config/ui_config.dart";
+import "../../../../theme/app_theme.dart";
+import "../../../../widgets/loading_widgets/scrolable_loader_builder.dart";
+import "../../../../widgets/loading_widgets/shimmer_loading.dart";
+import "sks_menu_header.dart";
+import "sks_menu_tiles.dart";
+
+class SksMenuViewLoading extends StatelessWidget {
+  const SksMenuViewLoading({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: context.colorTheme.whiteSoap,
+      body: const Shimmer(
+        linearGradient: shimmerGradient,
+        child: Column(
+          children: [
+            SksMenuHeaderLoading(),
+            Expanded(
+              child: _SksMenuTitleLoadingPaddings(),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _SksMenuTitleLoadingPaddings extends StatelessWidget {
```

Comment:
I don't like the name here. We don't use such convention anywhere else. Probably better to change it to `SksMenuTileLoading` and merge with existing widget of that name